scripts/asm-installer/tests/common.sh

```diff
@@ -10,6 +10,9 @@ WORKLOAD_NAME="vm"
 WORKLOAD_SERVICE_ACCOUNT=""
 INSTANCE_TEMPLATE_NAME=""
 SOURCE_INSTANCE_TEMPLATE_NAME="vm-source"
+CUSTOM_SOURCE_INSTANCE_TEMPLATE_NAME="custompublicvmsource"
+CUSTOM_IMAGE_LOCATION="us-central1-c"
+CUSTOM_IMAGE_NAME="testcustomimage"
```

Comment:
Just to confirm, this is a invalid image name, do we have tests that cover both invalid image name and valid image name?

Reply:
we test for custom,default and public image. invalid one will directly exit the bash